pulp_smash/tests/rpm/cli/test_langpacks.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+"""Tests that perform langpacks upload and removal."""
+from __future__ import unicode_literals
+
+import unittest2
+
+from pulp_smash import cli, config, utils
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+def get_langpacks_counts_by_repo_id(server_config, repo_id):
+    """Tell how many package langpacks are in a repository.
+
+    :param server_config: pulp_smash.config.ServerConfig server_config:
+        Information about the Pulp server being targeted.
+    :param repo_id: A RPM repository ID.
+    :returns: The number of package langpacks in a repository, as an ``int``.
+    """
+    keyword = 'Package Langpacks:'
+    completed_proc = cli.Client(server_config).run((
+        'pulp-admin repo list --repo-id {} '
+        '--fields content_unit_counts'
+    ).format(repo_id).split())
+    lines = [
+        line for line in completed_proc.stdout.splitlines()
+        if keyword in line
+    ]
+    assert len(lines) in (0, 1)
+    if len(lines) == 0:
+        return 0
+    else:
+        return int(lines[0].split(keyword)[1].strip())
+
+
+class UploadAndRemoveLangpacksTestCase(unittest2.TestCase):
+    """Upload langpacks and remove it through pulp-rpm.
+
+    This test targets `Pulp Smash #270`_. The test steps would be as follows:
+
+    1. upload langpacks to repo1
+    2. check whether the langpacks are really uploaded
+    3. remove all langpacks from repo1
+    4. check whether the langpacks are actually removed
+
+    .. _Pulp Smash #270: https://github.com/PulpQE/pulp-smash/issues/270
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a repository."""
+        cls.cfg = config.get_config()
+        cls.client = cli.Client(cls.cfg)
+        cls.repo_id = utils.uuid4()
+        cls.client.run(
+            'pulp-admin rpm repo create --repo-id {}'
+            .format(cls.repo_id).split()
+        )
+
+    def test_langpacks_upload_remove(self):
```

Comment:
OK..i see it. If they are still in one test method then SubTest is required...but now since they will be organized into two separate methods, just assertion would be fine. And unittest can continue all test methods even one might fail unexpectedly.